Leftover a11y warnings

e.g.

```
packages/components/histogram/src/HistogramG.svelte:
  357:5  warning  A11y: Elements with the 'button' interactive role must have a tabindex value  a11y-interactive-supports-focus
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -10,6 +10,15 @@
 	- moved `headerHeight` and `padding` to the `geometry` prop
 	- removed `fontSize`, using `glyphHeight` and `glyphWidth` in `geometry` instead
 - use `/ui`'s `Scroller`
+- resolved some a11y related warnings in `BarcharVDiv`
+
+## `@svizzle/histogram` v0.6.6 (next)
+
+- resolved some a11y related warnings in `HistogramDiv`
+
+## `@svizzle/legend` v0.4.4 (next)
+
+- resolved some a11y related warnings in `ColorBinsG`
 
 ## `@svizzle/time_region_value` v0.9.3 (next)
 
@@ -18,6 +27,7 @@
 ## `@svizzle/ui` v0.10.1 (next)
 
 - `ScreenSensor`: fixed the measured `glyph.height` being too big
+- `Banner`, `Switch` and `XorSelector`: resolved some a11y related warnings
 
 ## `@svizzle/site` v0.4.6 (next)
 

--- a/packages/components/barchart/CHANGELOG.md
+++ b/packages/components/barchart/CHANGELOG.md
@@ -6,7 +6,7 @@
 	- moved `headerHeight` and `padding` to the `geometry` prop
 	- removed `fontSize`, using `glyphHeight` and `glyphWidth` in `geometry` instead
 - use `/ui`'s `Scroller`
-
+- resolved some a11y related warnings in `BarcharVDiv`
 
 ## `@svizzle/barchart` v0.10.0
 

--- a/packages/components/histogram/CHANGELOG.md
+++ b/packages/components/histogram/CHANGELOG.md
@@ -1,3 +1,7 @@
+## `@svizzle/histogram` v0.6.6 (next)
+
+- resolved some a11y related warnings in `HistogramDiv`
+
 ## `@svizzle/histogram` v0.6.5
 
 - fixed `HistogramDiv` needing additional `overflow: hidden`

--- a/packages/components/histogram/src/HistogramG.svelte
+++ b/packages/components/histogram/src/HistogramG.svelte
@@ -362,6 +362,7 @@
 					on:click={resetSelection}
 					on:keydown={onKeyDown}
 					role='button'
+					tabindex='0'
 				/>
 			{/if}
 
@@ -401,6 +402,7 @@
 						>{displayValue}</text>
 
 						{#if flags.isInteractive}
+							<!-- FIXME: ARIA role set to 'none' pending solution -->
 							<rect
 								class='sensor'
 								height={barThickness}
@@ -409,6 +411,7 @@
 								on:mouseleave={onMouseleave(index)}
 								on:mousemove={isMousedown ? onMousemove(index) : null}
 								on:mouseup={onMouseup(index)}
+								role='none'
 								width={innerWidth}
 							/>
 						{/if}

--- a/packages/components/legend/CHANGELOG.md
+++ b/packages/components/legend/CHANGELOG.md
@@ -1,3 +1,7 @@
+## `@svizzle/legend` v0.4.4 (next)
+
+- resolved some a11y related warnings in `ColorBinsG`
+
 ## `@svizzle/legend` v0.4.3
 
 - updated deps

--- a/packages/components/legend/src/ColorBinsG.svelte
+++ b/packages/components/legend/src/ColorBinsG.svelte
@@ -341,15 +341,18 @@
 					on:click={resetSelection}
 					on:keydown={onKeyDown}
 					role='button'
+					tabindex='0'
 				/>
 			{/if}
 
 			<g transform='translate({origin.x},{origin.y})'>
 
 				<!-- bars -->
+				<!-- FIXME: ARIA role set to 'none' pending solution -->
 				<g
 					class='bars'
 					on:mouseleave={resetBrush}
+					role='none'
 				>
 					<rect
 						class='barsSensorBkg'
@@ -376,6 +379,7 @@
 								width={barWidth}
 							/>
 							{#if flags.isInteractive}
+								<!-- FIXME: ARIA role set to 'none' pending solution -->
 								<!-- svelte-ignore a11y-mouse-events-have-key-events -->
 								<rect
 									class='rectsensor'
@@ -385,6 +389,7 @@
 									on:mouseout={onMouseleave(index)}
 									on:mousemove={isMousedown ? onMousemove(index) : null}
 									on:mouseup={onMouseup(index)}
+									role='none'
 									width={barWidth}
 								/>
 							{/if}

--- a/packages/components/ui/CHANGELOG.md
+++ b/packages/components/ui/CHANGELOG.md
@@ -1,7 +1,7 @@
 ## `@svizzle/ui` v0.10.1 (next)
 
 - `ScreenSensor`: fixed the measured `glyph.height` being too big
-
+- `Banner`, `Switch` and `XorSelector`: resolved some a11y related warnings
 
 ## `@svizzle/ui` v0.10.0
 

--- a/packages/components/ui/src/Switch.svelte
+++ b/packages/components/ui/src/Switch.svelte
@@ -58,6 +58,7 @@
 		on:click={toggle}
 		on:keydown={onKeyDown}
 		role='button'
+		tabindex='0'
 	>
 		<span aria-hidden='true' class='bkg'></span>
 		<span aria-hidden='true' class='knob'></span>

